router/module.go

```diff
@@ -261,7 +262,7 @@ func (am AppModule) OnRecvPacket(ctx sdk.Context, packet channeltypes.Packet, re
 			prefixedDenom := transfertypes.GetDenomPrefix(packet.GetDestPort(), packet.GetDestChannel()) + newData.Denom
 			denom = transfertypes.ParseDenomTrace(prefixedDenom).IBCDenom()
 		}
-		unit, err := sdk.ParseUint(newData.Amount)
+		unit, err := math.ParseUint(newData.Amount)
```

Comment:
The rc moved this function into a separate `math` package which, as it appears, is released as a separate go module - `cosmossdk.io/math`.